source/ht_file.cpp

```diff
@@ -78,8 +78,7 @@ namespace Hatchit
         File::File(void)
             : IFile(),
             m_position(0),
-            m_size(0),
-            m_handle(nullptr) {}
+            m_size(0) {}
```

Comment:
We want to guarantee all of our objects are initialized when given an instance of whatever.  That way, we can guarantee the object is in a valid state whenever we receive one.